test/functional/eval/input_spec.lua

```diff
@@ -196,6 +228,19 @@ describe('input()', function()
     eq('Vim(call):E118: Too many arguments for function: input',
        exc_exec('call input("prompt> ", "default", "file", "extra")'))
   end)
+  it('supports highlighting', function()
+    feed([[:call input({"highlight": "RainBowParens"})<CR>]])
+    wait()
+    feed('(())')
+    wait()
```

Comment:
is it not just that the cmdline does not fit the screen width? If I make the terminal narrow I get the same behavior in TUI.